bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2145,7 +2145,10 @@ namespace bftEngine
 //				const SeqNum prevLastExecutedSeqNum = lastExecutedSeqNum;
 
 				lastExecutedSeqNum = newStateCheckpoint;
-                                metric_last_executed_seq_num_.Get().Set(lastExecutedSeqNum);
+                metric_last_executed_seq_num_.Get().Set(lastExecutedSeqNum);
```

Comment:
Nit: Any idea what's up with this indent change?

Reply:
it should be changed to spaces ASAP. All these tabs.. I tried to fix the indentation manually, just to avoid multiple format changes in the commit. Seems that I missed this one.

Reply:
I agree, I have also had issues with the tab indentation in this project before that I would not have had if it was space indentation.

Reply:
can you please change to spaces before merge?

Reply:
Switched to tabs since this file uses tabs over spaces